BLSQ/iaso
# Ia 2225 add create submission button

Add an action button in forms table to create submission

Related JIRA tickets : IA-2225

## Self proofreading checklist


## Changes

- Add A modal re-using `CreateReAssign` used for submissions, with a custom `IconButton`
- Pass ' dispatch` to `formsTableColumns`. I didn't refactor it as a hook, because it's still used with `SingleTable` so I'd rather change as little as possible to avoid creating unexpected bugs
- Updated cypress test

## How to test

- Run enketo locally
- Go to Forms
- If your user has the `submissions` permission, the table should include an action button to add a submission.
- click on it and try adding a submission
- Repeat with a user with insufficient permissions

## Print screen / video


https://github.com/BLSQ/iaso/assets/38907762/23888b2c-7e71-4cdd-a34e-ffb27ed2011d
